tests/testthat/test-view-composers.R

```diff
@@ -107,3 +107,17 @@ test_that("views are cached and retrieved", {
   expect_silent(initial.view %>% add_paraview(pos, 2, cached = TRUE))
   clear_cache()
 })
+
+test_that("add_paraview works with all distance families", {
+  expr <- generate_random_tibble(30, 5)
+  pos <- sample_grid_geometry(30, 10, 10)
+  initial.view <- create_initial_view(expr)
+  expect_error(misty.views <- initial.view %>% 
+                 add_paraview(pos, 2, family="bar"))
+  for (fam in c("gaussian", "exponential", "linear", "constant")) {
+    expect_message(misty.views <- initial.view %>% 
+                   add_paraview(pos, 2, family=fam),
+                   "Generating paraview")
+    expect_length(misty.views, 3)
```

Comment:
you can add all paraviews to the same view composition and expect_success then check for length

Reply:
I'd like to avoid that, since then I would have to use a different length parameter (otherwise the name of the paraview is not unique), what about something like that:
`families <- c("gaussian", "exponential", "linear", "constant")
  expect_message(test_run <- 
  purrr::map(families, ~ initial.view %>% add_paraview(family=.x, pos, 2)),
  "Generating paraview")
  purrr::walk(test_run, ~ expect_length(.x, 3))`